ZenPacks/zenoss/ZenPackLib/tests/test_impact_triggers.py

```diff
@@ -77,23 +67,30 @@ class ImpactTestCase(object):
     base: [SubComponent]
     monitoring_templates: [AuxComponent]
     impacted_by: [basicDevice, subComponents]
+device_classes:
+  /Server/Basic:
+    remove: true
+    zProperties:
+      zPythonClass: ZenPacks.zenoss.MockZenPack.BasicDevice
+
 """
 
 
-class TestImpactTriggers(ImpactTestCase):
+class TestImpactTriggers(ImpactTestCase, ZPLTestMockZenPack):
```

Comment:
What is the purpose of ImpactTestCase? It appears to be an empty class extending only `object` to me. Why have it at all?